schemas/entry.schema.json

```diff
@@ -80,11 +80,68 @@
       "additionalProperties": false,
       "properties": {
         "breakage": {
-          "description": "List of URLs to issues where actual site breakage is tracked. If available, point the link directly to a comment with clear STR",
+          "description": "List of issues tracking known site breakage.",
           "type": "array",
           "items": {
-            "type": "string",
-            "format": "uri"
+            "anyOf": [
+              {
+              "type": "string",
+              "format": "uri",
+              "description": "URL of tracking issue"
+            },
+              {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "URL of tracking issue"
+                },
+                "site": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "URL of broken site of page"
+                },
+                "platform": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "enum": ["all", "desktop", "mobile", "windows", "macos", "linux"],
+                    "description": "List of affected platforms. Default is 'all'"
+                  }
+                },
+                "last_reproduced": {
+                  "type": "string",
+                  "format": "date",
+                  "description": "Most recent date the issue was successfully reproduced"
+                },
+                "intervention": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "URL of intervention that is shipping or has been shipped. Link to the code in the GitHub repository, and use the canonical URLs to ensure persistance over time"
+                },
+                "impact": {
+                  "type": "string",
+                  "enum": ["site_broken", "feature_broken", "significant_visual", "minor_visual", "unsupported_message"],
+                  "description": "Type of breakage"
+                },
+                "affects_users": {
+                  "type": "string",
+                  "enum": ["all", "some", "few"],
+                  "description": "What fraction of users are affected. 'all' where any site user is likely to run into the issue, 'some' for issues that are common but many users will not experience, and 'few' where the breakage depends on an unusual configuration or similar."
+                },
+                "resolution": {
+                  "type": "string",
+                  "enum": ["site_changed", "site_fixed"],
```

Comment:
Should `site_broken` be also an option? (noticed it is used in one of the sidebar breakage reports)

Reply:
I thought `site_changed` could also cover "this site just doesn't exist anymore" because the use case I had in mind was tracking how often sites patch our issues directly vs how often they persist until the site undergoes larger changes; for the latter "broken" and "redesigned" are basically the same. But maybe there is some other case where we would benefit from tracking the difference?

Reply:
oops sorry - I thought `site_broken` is used in `resolution`, but it's actually one of the `impact` fields.
I guess my question is more about whether we want to track that site is still "broken" (i.e. issue reproducible) in the resolution field, or such issues will have no `resolution` field?

Reply:
I was assuming it would be easier to treat "still broken" as the missing value default rather than an explicit state.

Reply:
ok, that makes sense :)